valhalla/sif/dynamiccost.h

```diff
@@ -904,6 +907,10 @@ class DynamicCost {
     c.cost += service_penalty_ *
               (edge->use() == baldr::Use::kServiceRoad && pred->use() != baldr::Use::kServiceRoad);
     // shortest ignores any penalties in favor of path length
+
+    c.cost += service_penalty_ *
```

Comment:
Why are we adding `service_penalty_` twice?

Reply:
@mandeepsandhu merge issue.  Thank you for seeing this one

Reply:
updated.

Reply:
Since this affected the base transition cost, should we RAD test a few routes to see the result is still as expected? Although if this was only due to a recent merge and the RAD results in PR were performed before this change, then we can skip it. I'll leave it to you